Dockerfile

```diff
@@ -26,6 +26,8 @@ RUN apt-get update \
        unzip \
        git \
        gradle \
+       zipalign \
+       awscli \
```

Comment:
This is not generic package for use this image.
You can customize your image by fork this repository and add your package do you want.

Thank you.

Reply:
I agree about awscli; however zipalign is basically a must for signing [ionic] android apk builds.

Reply:
Ok. I got it.

Thank you.